### ISSUE TITLE
App: ArrayIndexOutOfBoundsException in MacUtil.getApps

### ISSUE THREAD

**Issue by Torbilicious** [RaiMan/SikuliX1#310]
Simply calling new `App("path").open()` on my system yields an ArrayIndexOutOfBoundsException.  

Stacktrace:

```
java.lang.ArrayIndexOutOfBoundsException: 1

	at org.sikuli.natives.MacUtil.getApps(MacUtil.java:286)
	at org.sikuli.natives.MacUtil.get(MacUtil.java:84)
	at org.sikuli.script.App.isRunning(App.java:554)
	at org.sikuli.script.App.openAndWait(App.java:627)
	at org.sikuli.script.App.open(App.java:609)

```

When the error occurs the process variable has the content ` |||, 338, Finder, Finder, alias Mojave:System:Library:CoreServices:Finder.app:`, which does not match the given regex `process.split(", \\|\\|\\|,");`.

I am happy to provide any additional info if necessary.

OS: macOS 10.14.6 
Sikulix version: 2.0.2
Java: Oracle JDK 8

**Comment by Torbilicious** [comment]
I debugged a bit further and found out, that the result from running the AppleScript looks like:  
```
, |||, 338, Finder, Finder, alias Mojave:System:Library:CoreServices:Finder.app:, ###
, , |||, 470, Spotify, Spotify, alias Mojave:Applications:Spotify.app:, ###
, , |||, 552, iTerm2, iTerm2, alias Mojave:Applications:iTerm.app:, ###
```

```java
if (process.startsWith(", ")) {
    process = process.substring(2);
}
```

Now removes thebeginning of the line which was used in the regex mentioned above.  
Debugging why the Applescript returns this is a bit out of my comfort zone :)

**Comment by RaiMan** [comment]
Thanks, will check.

**Comment by RaiMan** [comment]
fixed with branch::release_2.0.x::commit https://github.com/RaiMan/SikuliX1/commit/d5e55899d92e03231348225d96c4031b8462fd3c

available in 2.0.3
